xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexAccountServiceRaw.java

```diff
@@ -130,7 +156,10 @@ public String withdraw(
             tagOrPaymentId);
     req.setCurrency(currency);
     BitfinexWithdrawalResponse[] withdrawResponse =
-        bitfinex.withdraw(apiKey, payloadCreator, signatureCreator, req);
+        decorateApiCall(() -> bitfinex.withdraw(apiKey, payloadCreator, signatureCreator, req))
+            .withRetry(retry("account-withdraw"))
```

Comment:
is this operation idempotent?

if not then doing a retry may result in withdrawing more fund then you wonted